apps/console/src/features/organizations/pages/organizations.tsx

```diff
@@ -101,6 +101,8 @@ const OrganizationsPage: FunctionComponent<OrganizationsPageInterface> = (
     const [ organization, setOrganization ] = useState<OrganizationResponseInterface>(null);
     const [ after, setAfter ] = useState<string>("");
     const [ before, setBefore ] = useState<string>("");
+    const [ authorizedListBefore, setAuthorizedListBefore ] = useState<string>("");
```

Comment:
Why do we use "List" in the variable name if they are strings?

Reply:
const {
        data: authorizedOrganizationList,
        isLoading: isAuthorizedOrganizationListRequestLoading,
        error: authorizedListFetchRequestError,
        mutate: updateAuthorizedList
    } = useAuthorizedOrganizationsList(
        filterQuery,
        listItemLimit,
        authorizedListAfter,
        authorizedListBefore,
        false
    );
    
Here I used, authorizedListAfter and authorizedListBefore values as before and after for this API call. Therefore, used that variable name.

Reply:
This variables are renamed to `authorizedListPrevCursor` and `authorizedListNextCursor` as discussed.